Remote MCP documentation

bryantgillespie opened the pull request.

bryantgillespie committed:
update nuxt ui

bryantgillespie committed:
add chat content component

bryantgillespie committed:
fix PH capture for CLI snippet

bryantgillespie committed:
scaffold out docs structure

bryantgillespie committed:
fix links

bryantgillespie committed:
add config variables

bryantgillespie committed:
fix broken links

bryantgillespie committed:
show sample tool calls

bryantgillespie committed:
fix ids

bryantgillespie committed:
updates

bryantgillespie committed:
fix broken links

bryantgillespie committed:
fix broken link

bryantgillespie committed:
update prompt doc

bryantgillespie committed:
fix more links

bryantgillespie committed:
refine security guidelines for MCP usage and compliance considerations

bryantgillespie committed:
add linkChecker configuration to skip absolute site URL inspections

bryantgillespie committed:
disable auto scroll

ComfortablyCoding commented on content/guides/10.ai/1.mcp/0.index.md:
```diff
@@ -1,181 +1,235 @@
-title: Installation
-description: The Directus Content MCP Server allows you to interact with your Directus data through AI tools using the Model Context Protocol.
+title: Overview
+description: Connect AI assistants directly to your Directus instance. Let Claude, ChatGPT, and other AI tools read and write your content without manual copy-pasting.
```
```suggestion
description: Connect AI assistants directly to your Directus instance. Let Claude, ChatGPT, and other AI tools manage your content without manual copy-pasting.
```

ComfortablyCoding commented on content/guides/10.ai/1.mcp/0.index.md:
```diff
@@ -1,181 +1,235 @@
+**MCP is currently in Beta and requires Directus v11.12 +**. For older versions, use the [Local MCP alternative](/guides/ai/mcp/local-mcp).
+::
+
+## Is this for you?
```
```suggestion
## How can this help me?
```

ComfortablyCoding replied:
This also seems to be a shortened duplicated of the `what-you-can-do-with-the-directus-mcp-server` section

ComfortablyCoding commented on content/guides/10.ai/1.mcp/0.index.md:
```diff
@@ -1,181 +1,235 @@
+## How it works
+
+1. Enable MCP in Directus Settings → AI
+2. Create access token for your MCP user (we recommend creating a dedicated user for MCP operations)
```
```suggestion
2. Generate an access token for your MCP user (we recommend creating a dedicated user for MCP operations)
```

ComfortablyCoding commented on content/guides/10.ai/1.mcp/1.installation.md:
```diff
@@ -0,0 +1,285 @@
+::::
+
+1. Navigate to **User Directory**
+2. Find yourexisting user
```
```suggestion
2. Find your existing user
```

bryantgillespie committed:
security updates

bryantgillespie committed:
fix typo

bryantgillespie committed:
Update content/guides/10.ai/1.mcp/0.index.md

Co-authored-by: daedalus <44623501+ComfortablyCoding@users.noreply.github.com>

bryantgillespie committed:
Update content/guides/10.ai/1.mcp/0.index.md

Co-authored-by: daedalus <44623501+ComfortablyCoding@users.noreply.github.com>

bryantgillespie committed:
Update content/guides/10.ai/1.mcp/0.index.md

Co-authored-by: daedalus <44623501+ComfortablyCoding@users.noreply.github.com>

bryantgillespie committed:
add optin text

bryantgillespie committed:
add security notice

Commit:
Update installation guide with LLM integration details (#454)

Added information about controlling LLM integration and connecting to AI tools.

bryantgillespie committed:
troubleshooting

bryantgillespie committed:
Merge branch 'main' into remote-mcp-docs
